demo/monkey-patching/index.php

```diff
@@ -4,19 +4,18 @@
 
 use BetterReflection\Reflector\ClassReflector;
 use BetterReflection\SourceLocator\Type\SingleFileSourceLocator;
-use PhpParser\PrettyPrinter\Standard as CodePrinter;
+use BetterReflection\Util\Autoload;
+
+Autoload::initialise();
 
 // Create the reflection first (without loading)
 $classInfo = (new ClassReflector(new SingleFileSourceLocator(__DIR__ . '/MyClass.php')))->reflect('MyClass');
+Autoload::addClass($classInfo);
```

Comment:
IMO, the autoloader shouldn't be static. We can add a way to get a singleton reference to it (it is a singleton by PHP's design, after all), but not a direct static method (Larval "facade-non-facade" approach)